lib/loaders/gravity.js

```diff
@@ -15,7 +15,10 @@ load.with = (accessToken) => {
   const authenticatedGravityLoader = authenticatedHttpLoader(gravity, accessToken);
   return (path, options = {}) => {
     const key = toKey(path, options);
-    return authenticatedGravityLoader(key, accessToken);
+    if (accessToken) {
```

Comment:
This is the main change, if you are using `with` and don't have an access token, we want to fall back to the standard cached loader.